OxfordBodleian/Juel-Jensen/BDLaethf30.xml

```diff
@@ -136,24 +136,40 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         </item>
                                     </list>
                                 </collation>
-                                <condition key="deficient">The manuscript is damaged by humidity, in particular at the beginning
-                                    and at the end. The sewing has become partly undone. 
+                                <condition key="deficient">
+                                  The manuscript is in a poor state of preservation.
+                                  <locus target="#i"/> is much less broad than the other leaves, but this might be its original state.
+                                  The manuscript is damaged by humidity, in particular at the beginning
+                                  and at the end. The lower margin is darkened by use. The ink is occasionally smudged, e.g. <locus target="#41v #104v"/>.
+                                  The leaves present stains of various origins, uneven deformations, tears, and losses,
+                                  which sometimes result in more or less extensive losses of text, e.g. <locus target="#55 #150 #152"/>.
+                                  Dark red stains on <locus from="20" to="23"/> and <locus target="#115"/>.
+                                  <!--the stains could be the result of oxidation of the parchment support
```

Comment:
In my view we can also incorporate this comment in the text, for example "Dark red stains on ..., possibly due to oxidation of the support"